include/z64camera.h

```diff
@@ -1060,6 +1060,33 @@ typedef struct {
     { fov, CAM_DATA_FOV }, \
     { flags, CAM_DATA_FLAGS }
 
+typedef enum {
+    /*  0x1 */ ONEPOINT_CS_ACTION_1 = 1,
+    /*  0x2 */ ONEPOINT_CS_ACTION_2,
+    /*  0x3 */ ONEPOINT_CS_ACTION_3,
+    /*  0x4 */ ONEPOINT_CS_ACTION_4,
+    /*  0x9 */ ONEPOINT_CS_ACTION_9 = 9,
+    /*  0xA */ ONEPOINT_CS_ACTION_10,
+    /*  0xB */ ONEPOINT_CS_ACTION_11,
+    /*  0xC */ ONEPOINT_CS_ACTION_12,
+    /*  0xD */ ONEPOINT_CS_ACTION_13,
+    /*  0xF */ ONEPOINT_CS_ACTION_15 = 15,
+    /* 0x10 */ ONEPOINT_CS_ACTION_16,
+    /* 0x11 */ ONEPOINT_CS_ACTION_17,
+    /* 0x12 */ ONEPOINT_CS_ACTION_18,
+    /* 0x13 */ ONEPOINT_CS_ACTION_19,
+    /* 0x15 */ ONEPOINT_CS_ACTION_21 = 21,
+    /* 0x18 */ ONEPOINT_CS_ACTION_24 = 24
+} OnePointCsAction;
+
+#define ONEPOINT_CS_ACTION_40 0x40
+#define ONEPOINT_CS_ACTION_BGCHECK 0x80
```

Comment:
Probably this will change later but even for now maybe there should be a stronger difference in names between the enum names and macros names

for example (idk)

```suggestion
    /* 0x15 */ ONEPOINT_CS_ACTION_ID_21 = 21,
    /* 0x18 */ ONEPOINT_CS_ACTION_ID_24 = 24
} OnePointCsAction;

#define ONEPOINT_CS_ACTION_40 0x40
#define ONEPOINT_CS_ACTION_BGCHECK 0x80
```

or

```suggestion
    /* 0x15 */ ONEPOINT_CS_ACTION_21 = 21,
    /* 0x18 */ ONEPOINT_CS_ACTION_24 = 24
} OnePointCsAction;

#define ONEPOINT_CS_ACTION_FLAG_40 0x40
#define ONEPOINT_CS_ACTION_FLAG_BGCHECK 0x80
```

or both

Reply:
Works for me. I see these being named better as the system is better understood from a bigger-picture point of view, and are more placeholders for now anyway